RELEASE-NOTES.md

```diff
@@ -22,7 +22,9 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 ### Deprecated
 
 ### Changed
-- Updated Umpire TPL to v2.0.0, which, natively handles zero byte re-allocations consistently. Workaround
+- Updated Conduit to v0.5.1
+- Updated Axom to v0.10.1
```

Comment:
I think this should be RAJA- 0.11.0

Reply:
Yup, definitely, not sure how that happened...